webapp/src/main/java/life/qbic/views/login/LoginHandler.java

```diff
@@ -0,0 +1,48 @@
+package life.qbic.views.login;
+
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.notification.Notification;
+import life.qbic.usermanagement.persistence.UserJpaRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LoginHandler implements LoginHandlerInterface {
+
+    private final UserJpaRepository userRepository;
+
+    private LoginLayout registeredLoginView;
+
+    LoginHandler(@Autowired UserJpaRepository repository){
+        this.userRepository = repository;
+    }
+
+    @Override
+    public boolean register(LoginLayout loginView) {
+        if (registeredLoginView != loginView) {
+            this.registeredLoginView = loginView;
+            // orchestrate view
+            addListener();
```

Comment:
```suggestion
            registeredLoginView = loginView;
            // orchestrate view
            addListener();
```

or 

```suggestion
            this.registeredLoginView = loginView;
            // orchestrate view
            this.addListener();
```

to be consistent